.ci/docker-setup.sh

```diff
@@ -48,10 +48,10 @@ if [ "$ELASTIC_STACK_VERSION" ]; then
 
     if [ "$INTEGRATION" == "true" ]; then
         docker-compose down
-        docker-compose build
+        docker-compose build --quiet
```

Comment:
we are getting ~34k log lines with `docker-compose build` command. Adding `--quiet` to see if it decreases.

Reply:
FYI: now `.ci/docker-setup.sh` log lines about 20 lines.